Source/Orts.Simulation/Simulation/Activity.cs

```diff
@@ -1388,12 +1388,11 @@ private Train matchesConsistNoOrder(List<string> wagonIdList)
         {
             foreach (var trainItem in Simulator.Trains)
             {
-                bool lEngine = false;
+                //bool lEngine = false;
```

Comment:
It's bad form to litter the source code with commented code sections. Just delete it.
```suggestion
```